specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2024-05-15/examples/services/sqldedicatedgateway/CosmosDBSqlDedicatedGatewayServiceCreate.json

```diff
@@ -0,0 +1,44 @@
+{
+  "parameters": {
+    "accountName": "ddb1",
+    "resourceGroupName": "rg1",
+    "api-version": "2024-05-15",
+    "subscriptionId": "subid",
+    "serviceName": "SqlDedicatedGateway",
+    "createUpdateParameters": {
+      "properties": {
+        "instanceSize": "Cosmos.D4s",
+        "instanceCount": 1,
+        "serviceType": "SqlDedicatedGateway",
+        "dedicatedGatewayType": "IntegratedCache"
+      }
+    }
+  },
+  "responses": {
+    "202": {},
+    "200": {
```

Comment:
>200 [](http://example.com/codeflow?start=5&length=3)

the example is called create- but shouldn't _create_ be responding with __201__?

Reply:
This API was conceptualized before the current ARM guidelines, and returns 200. It's not an ARM tracked resource so shouldn't be an issue for now.